Casks/vienna.rb

```diff
@@ -1,6 +1,6 @@
 cask "vienna" do
-  version "3.6.2,7371"
-  sha256 "c43487411c8f39d29e82f0cc618a04d00a7147afa593dfe4cd45de2be5280bc6"
+  version "3.7.0"
```

Comment:
The short `sparkle:version` from the actual feed is required:

```suggestion
  version "3.7.0,7429"
```

Reply:
I am not convinced by the change introduced by PR #96686
Vienna versioning scheme never referred to the sparkle:version data. Cf. the [releases page](https://github.com/ViennaRSS/vienna-rss/releases) and previous PR #94328 or #94480.

If for some reason, it is deemed necessary to add data to the official numbering, I think it should be the commit short SHA (in this case `:5698f89d:`) not the commit sequence (7371), as this later might create confusions when dealing with different branches (Vienna has `master`, `stable` and `old stable` branches)

Reply:
Perhaps @reitermarkus can explain a bit about it since it's new to me, but for now, *not* making the change means that either the CI fails or nobody updates vienna.